apps/analysis/tests/test_mutations.py

```diff
@@ -392,9 +419,11 @@ def test_automatic_summary(self, trigger_results_mock, RequestHelperMock):
         lead1 = LeadFactory.create(project=self.project)
         lead2 = LeadFactory.create(project=self.project)
         another_lead = LeadFactory.create(project=self.another_project)
+        lead3 = LeadFactory.create(project=self.private_project)
```

Comment:
Let's create another confidential leads RESTRICTED, CONFIDENTIAL and some entries. Make sure the generated files doesn't include those entries, even if filters allows.

Similar to what we did before here https://github.com/the-deep/server/pull/1502/files#diff-223594694777faf05fef2c6acee1587a281db5d1002813ca7b0e43862d6feda0R256-R268

Reply:
For this one, Let's merge the Mentioned PR and Update the fields validation. They are on same function that's why.

Reply:
@AdityaKhatri Will merge this https://github.com/the-deep/server/pull/1502. After that, let's confirm the above changes.